tests/syscall/mount/mount-immutables.bats

```diff
@@ -0,0 +1,596 @@
+#!/usr/bin/env bats
+
+#
+# Verify trapping & emulation on "mount" and "unmount2" syscalls
+#
+
+load ../../helpers/run
+load ../../helpers/syscall
+load ../../helpers/docker
+load ../../helpers/environment
+load ../../helpers/mounts
+load ../../helpers/sysbox-health
+
+skipTest=0
```

Comment:
Why do we need this variable?

We already have a "run_only_test <name>" function that we can add to `function setup()` to skip tests.